packages/devextreme/js/ui/tabs.js

```diff
@@ -346,13 +357,30 @@ const Tabs = CollectionWidget.inherit({
         return this._$wrapper;
     },
 
-    _renderScrollable: function() {
-        const $itemContainer = this.$element().wrapInner($('<div>').addClass(TABS_SCROLLABLE_CLASS)).children();
+    _getScrollableDirection() {
         const isVertical = this._isVertical();
-        const scrollableDirection = isVertical ? 'vertical' : 'horizontal';
```

Comment:
why don't use this._isVertical() directly here? in this case isVertical const is not needed. and this const doesn't make the code more readable

Reply:
Because this way is better for debugging.